test/src/unit-vfs.cc

```diff
@@ -46,11 +46,9 @@ using namespace tiledb::test;
 TEST_CASE("VFS: Test read batching", "[vfs]") {
   ThreadPool compute_tp(4);
   ThreadPool io_tp(4);
-
   URI testfile("vfs_unit_test_data");
-  std::unique_ptr<VFS> vfs(new VFS);
-  REQUIRE(
-      vfs->init(&g_helper_stats, &compute_tp, &io_tp, nullptr, nullptr).ok());
+  std::unique_ptr<VFS> vfs(
+      new VFS(&g_helper_stats, &compute_tp, &io_tp, Config{}));
```

Comment:
Is the allocation still necessary here? It could remain without making a test defective, but seems a bad example for others.

Reply:
Changed in #3523